### ISSUE TITLE
Questions About Reproducing the SceneScript Results Reported in the Paper

### ISSUE THREAD

**Issue by Infinity12306** [manycore-research/SpatialLM#135]
Hi, thank you for releasing the code and models for this work.

I am currently trying to run SceneScript and reproduce the results reported in the paper. However, I have encountered a few issues and would appreciate your clarification:

Two SceneScript checkpoints appear to be available: one trained on ASE and another trained on a proprietary dataset. Which checkpoint did you fine-tune and use for the comparisons reported in your paper?

I am unable to download the released SceneScript model weights from the official project page:

https://www.projectaria.com/scenescript/#/

When I click the download button highlighted by the red rectangle in the screenshot below, nothing happens. Did you encounter the same issue? Could you please share how you obtained the model weights, or provide an alternative download link?

<img width="3378" height="1956" alt="SceneScript model download issue" src="https://github.com/user-attachments/assets/0ec90459-2781-409b-a446-99c6e4ffef73" />

Thank you for your help.

**Comment by Infinity12306** [comment]
Additionally, I would like to clarify the exact ScanNet split used in your experiments. The official ScanNet split contains 1,201 training scenes and 312 validation scenes. Did you use the 1,201 training scenes for fine-tuning and the 312 validation scenes for evaluation?

Also, did you reserve a separate validation set for checkpoint selection, or did you use the evaluation set to select the best checkpoint?

**Comment by SamMaoYS** [comment]
Hi @Infinity12306. We used the checkpoint trained on the ASE dataset and fine-tuned it for the different evaluation tasks. We did not encounter any issues when downloading the checkpoints on our end. Hopefully the issue will be resolved soon. We have no plan to provide alternative download links. Thank you for your understanding.

Regarding the ScanNet training, yes, we used the original train/validation split you mentioned above. We selected the final checkpoint primarily based on the convergence of the training loss, and the performance on a small set of randomly held-out training samples (approximately a dozen) for evaluation.

**Comment by Infinity12306** [comment]
Thank you for all the valuable implementation details!!! Just one last question, did you download the model weights from the same website as shown in the image?

**Comment by SamMaoYS** [comment]
Yes, i downloaded from the same website you shared.
